information/licensing/Concurrent_sessions_based_licensing.md

```diff
@@ -0,0 +1,66 @@
+# Concurrent Sessions-Based Licensing
+
+Instances of @@name which are registered and paid by the subscription plans **Small Business Basic** and **Small Business Advanced** are licensed under the **Concurrent Sessions-Based** licensing.
+This type of license allows the client to run a **number of active sessions** at the same time.
+
+## Single Concurrent Session Definition
+
+A **single concurrent session** for an @@name instance is an active connection established between **a single desktop application**, or **web** or **mobile client**, on a single user's **device**, on one side, and the **servers** on which the @@name system's instance and the user database are present and operate, on the other.  
+
+This session grants access to the system and the database for this user via this single application or web client on a desktop computer, laptop, or mobile device (a smartphone, tablet, or other).
+
+### Example of a Singe Concurrent Session
+
+An example of the former would be if you start **a single instance of your desktop application** OR **a single instance of your web client**, and then use it to connect to your @@name instance covered by your subscription plan.  
+
+If no *specific conditions* apply, this would be deemed **a single active and concurrent session** between your user's application and the server.
+
+## Multiple Concurrent Sessions Definition
+
+**Multiple concurrent sessions** for an @@name instance occur when multiple active connections between **applications** and/or **web clients** are established to the **server** the @@name instance is present and is running on.  
+
+These sessions allow our clients to execute, control, manage, and oversee their business processes from multiple locations, devices, and by multiple people and automated applications simultaneously.
```

Comment:
Avoid talking about "us", unless absolutely necessary. Concentrate on the user.

Reply:
Regarding remarks: 

Avoid talking about "us", unless absolutely necessary. Concentrate on the user. - Stated in a different way, addressing the user/client instead of "our clients".